### PR TITLE
Fix componenent

### PR DESCRIPTION
**Pull request by jksmall0631**
Spent a lot of time refactoring favorites render functionality only to find that the problem was in the spelling of component. We reverted back to this component structure so that we can render a single component that renders favorites based on a condition rather than having an additional 4+ components for all movies and favorite movies.

### CODE REVIEW

**Comment by devinmarieb** [comment]
that was an hour well spent....

File: app/components/MovieIndex.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router'
 
+ // ({this.props.movieReducer, this.props.userSignInReducer}) =>
 
-const MovieIndex = ({movieReducer, userSignInReducer}) => {
+class MovieIndex extends Component{
```
Comment (stephanieea) [inline app/components/MovieIndex.js:6]: YES!

File: app/components/MovieIndex.js [hunk 2]
```diff
@@ -15,26 +19,29 @@ const MovieIndex = ({movieReducer, userSignInReducer}) => {
...
-        {userSignInReducer.user ? <button className='favs' onClick={()=> showFavorites(userSignInReducer.id)}> Show Favorites </button> : ''}
-      </Link>
-    {movie}
-    </div>
-  )
-}
+    return (
+      <div className='movie-container'>
+        <Link to={'/favorites'} >
+          {this.props.userSignInReducer.user ? <button className='favs' onClick={() => this.props.switchToFavs()}> Show Favorites </button> : ''}
```
Comment (stephanieea) [inline app/components/MovieIndex.js:40]: Cool!

File: app/reducers/reducer-movieIndex.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const movieIndexReducer = (state = [], action) => {
+  switch(action.type){
+    case 'TOGGLE_FAVS':
```
Comment (stephanieea) [inline app/reducers/reducer-movieIndex.js:3]: Good Naming!

File: app/reducers/reducer-userSignIn.js [hunk 4]
```diff
@@ -1,12 +1,10 @@
 const userSignInReducer = (state =[], action) => {
   switch(action.type){
     case 'USER_LOGIN':
-    console.log(action)
     return Object.assign({}, state, {user: action})
   }
   switch(action.type){
     case 'ADD_FAVS':
```
Comment (stephanieea) [inline app/reducers/reducer-userSignIn.js:7]: We might want to consider changing this name

**Pull request merged**